mantidproject/mantid
# Reorganise unit tests of fitting functions:: 'General' Category - Chunk 3

**Description of work.**

All unit tests using `Fit` algorithm from General Category have been moved to `FitTest.h`. Most of the files mentioned in the issue were not utilising the `Fit` algorithm. 

**To test:**
Check all build pass
Check all test pass locally without any issue
Check that the following files are not use the `Fit` algorithm
- ReflectivityMulf 
- TabulatedFunction
- ThermalNeutronBk2BkExpAlpha
- ThermalNeutronBk2BkExpBet
- ThermalNeutronBk2BkExpConvPVoigt



_Does not need to be in the release notes._
Fixes #16485



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
